### PR TITLE
fix(styles): update grid-gap to column-gap for inline dropdown

### PR DESCRIPTION
**Pull request by joshblack**
This came from a report in Slack. Currently, our inline dropdown uses `grid-gap` which adds both row and column spacing. This PR switches it to use `column-gap` so that only column spacing is used.

image

#### Changelog

**New**

**Changed**

- Update styles for dropdown so that only column spacing is used

**Removed**

#### Testing / Reviewing

- View the deploy preview and verify that the row spacing as shown in the screenshot above no longer exists

### CODE REVIEW

**Review by tay1orjones — APPROVED** [review]

**Review by jnm2377 — APPROVED** [review]

**Review by jnm2377 — CHANGES_REQUESTED** [review]
ughh... wait... after inspecting, I'm realizing it doesn't actually change the spacing. If I add `column-gap: 0` specifically, it does.

Screen Shot 2021-09-29 at 11 17 10 AM
Screen Shot 2021-09-29 at 11 17 20 AM

File: packages/components/src/components/dropdown/_dropdown.scss [hunk 1]
```diff
@@ -24,7 +24,7 @@
   .#{$prefix}--dropdown__wrapper--inline {
     display: inline-grid;
     align-items: center;
-    grid-gap: rem(24px);
+    grid-gap: 0 rem(24px);
```
Comment (jnm2377) [inline packages/components/src/components/dropdown/_dropdown.scss:27]: I think this is backwards. Should it be?
```suggestion
    grid-gap: rem(24px) 0;
```

File: packages/styles/scss/components/dropdown/_dropdown.scss [hunk 2]
```diff
@@ -25,7 +25,7 @@
   .#{$prefix}--dropdown__wrapper--inline {
     display: inline-grid;
     align-items: center;
-    grid-gap: rem(24px);
+    grid-gap: 0 rem(24px);
```
Comment (jnm2377) [inline packages/styles/scss/components/dropdown/_dropdown.scss:28]: same as above

File: packages/components/src/components/dropdown/_dropdown.scss (hunk 1 above)
Comment (joshblack) [reply packages/components/src/components/dropdown/_dropdown.scss:27]: @jnm2377 I think the syntax for gap is `<row-gap> <column-gap>` and we are setting 0 for the row gap to remove the vertical spacing caused by the listbox styles. Let me know if I'm misunderstanding 👀

**Comment by jnm2377** [comment]
wait... is the column spacing the vertical spacing or the horizontal spacing? 😭 Now I'm confused lol

**Comment by joshblack** [comment]
@jnm2377 haha, totally get that. I believe that column is horizontal and row is vertical spacing.

File: packages/components/src/components/dropdown/_dropdown.scss (hunk 1 above)
Comment (jnm2377) [reply packages/components/src/components/dropdown/_dropdown.scss:27]: ah ok ok. I think I was misunderstanding which one we were removing! I thought we were removing column gap. Gotcha gotcha.

**Review by jnm2377 — APPROVED** [review]

**Pull request merged**